coala_quickstart/green_mode/QuickstartBear.py

```diff
@@ -0,0 +1,31 @@
+from coalib.bears.LocalBear import LocalBear
+
+
+class QuickstartBear(LocalBear):
+    """
+    A bear to parse out the file_dict and return
+    values of settings that can take an infinite
+    set of values and can be determined easily
+    by looking at the file contents.
+    """
+
+    def run(self, filename, file):
+        """
+        :return:
+            A dict of setting name as key and the appropriate
```

Comment:
You're returning a `list` of `dict` below.

Reply:
:+1: